examples/salesforce_sink_example.py

```diff
@@ -83,7 +83,7 @@ def example_1_rate_source_to_accounts():
         
         # Write to Salesforce
         query = account_data.writeStream \
-            .format("salesforce") \
+            .format("salesforce-sink") \
```

Comment:
To follow the naming convention for Scala data source (e.g `org.spark.sql.<datasource>`) can we use something like: `pyspark.datasource.salesforce`? Or we can just use `sfdc` (abbreviation for salesforce).